seqr/views/apis/summary_data_api_tests.py

```diff
@@ -134,6 +134,11 @@ def test_saved_variants_page(self):
         expected_variant_guids.add('SV0000002_1248367227_r0390_100')
         self.assertSetEqual(set(response.json()['savedVariantsByGuid'].keys()), expected_variant_guids)
 
+        multi_tag_url = reverse(saved_variants_page, args=['Review;Tier 1 - Novel gene and phenotype'])
+        response = self.client.get('{}?gene=ENSG00000135953'.format(multi_tag_url))
+        self.assertEqual(response.status_code, 200)
+        self.assertSetEqual(set(response.json()['savedVariantsByGuid'].keys()), {'SV0000001_2103343353_r0390_100'})
```

Comment:
use a constant for this guid

Reply:
It is in a test assertion. Are you sure a constant is better than a string?

Reply:
I was thinking since its reused, because its in the single tag filtering and the multi tag filtering, its better to share a constant. But if you prefer for consistency with the test file, we can leave it